shared/desktop/yarn-helper/log-to-trace.tsx

```diff
@@ -12,11 +12,11 @@ const fs = require('fs')
 const moment = require('moment')
 
 // core regs
-const reg = /([^ ]+) ▶ \[DEBU (keybase|kbfs) ([^:]+):(\d+)] ([0-9a-f]+) ([^[]+)(\[tags:([^\]]+)])?/
+const reg = /([^ ]+) ▶ \[DEBU (keybase|kbfs) ([^:]+):(\d+)] ([0-9a-f]+) ([^[]+)(.*)?/
 const tagsReg = /\[tags:([^\]]+)]/
 const methodPrefixReg = /^(\+\+Chat: )?/
 const methodResultReg = / -> .*$/
-const typeAndMethodReg = /^(\W*)/
+const typeAndMethodReg = /^(gui|[+-|])/
```

Comment:
The `[+-|]` captures all characters between character codes 43 (+) to 124 (|) because the `-` makes in , not just `+`, `-`, and `|` like it looks like you want to. Escaping the `-` will just match the `+`, `-`, and `|`. For easy copy/paste: `^(gui|[+\-|])`

Reply:
updated